src/components/PopupWithConfirmation.js

```diff
@@ -1,6 +1,6 @@
 import Popup from "./Popup.js";
 
-export default class PopupCardDeleteVerify extends Popup {
+export default class PopupWithConfirmation extends Popup {
   constructor(popupSelector, handleFormSubmit) {
```

Comment:
`PopupWithConfirmation` is a general class. It can be used for any confirmation, not only card delete confirmation. Please remove all the mentions of `card` in this file

Reply:
You should remove  all the mentions of cards in this file